<ui version="4.0" >
 <class>LaPlantManagerBase</class>
 <widget class="QDialog" name="LaPlantManagerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plant Profile Manager</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBoxPlantDescription" >
     <property name="title" >
      <string>Plant Description</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="1" colspan="3" >
       <widget class="QComboBox" name="comboBoxYieldUnits" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">1 dunum = 1000 square meters&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">1 hectare = 10,000 square meters&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <item>
         <property name="text" >
          <string>* Per Dunum</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>* Per Hectare</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="labelPlantDescription6" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Fodder produced from Wheat&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Fodder is the straw and chaff left after harvesting the grain.  Enter the number of Kilograms expected for this crop per (either) dunum or hectare&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Fodder</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="spinBoxCropFodderProduction" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Fodder produced from Wheat&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Fodder is the straw and chaff left after harvesting the grain.  Enter the number of Kilograms expected to be produced from this crop per (either) dunum or hectare.  &lt;span style=" font-weight:600; color:#0000ff;">Note that this fodder amount is solely for consumption by animals.&lt;/span>  If straw or chaff is being used for other purposes, it must be taken out of the total expected fodder level.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="suffix" >
         <string> Kg*</string>
        </property>
        <property name="maximum" >
         <number>400</number>
        </property>
        <property name="value" >
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Food Value</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="spinBoxCropFodderCalories" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Calories per kilogram in WHEAT fodder&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Food value of fodder when used as feed for domestic animals.  This is measured in calories per kilogram.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="suffix" >
         <string> Cal/Kg</string>
        </property>
        <property name="maximum" >
         <number>5000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Food Value</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spinBoxCropYield" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Yield of Crop&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="suffix" >
         <string> Kg*</string>
        </property>
        <property name="maximum" >
         <number>400</number>
        </property>
        <property name="value" >
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelPlantDescription4" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Crop Yield</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QSpinBox" name="spinBoxCropCalories" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix" >
         <string> Cal/Kg</string>
        </property>
        <property name="maximum" >
         <number>5000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="leDescription" >
        <property name="whatsThis" >
         <string>Description of the current layerset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QLineEdit" name="leName" >
        <property name="whatsThis" >
         <string>Short name for the current layerset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Area Units</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="grpProfiles" >
     <property name="title" >
      <string>Available Plant Profiles</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="6" >
       <widget class="QTableWidget" name="tblPlants" >
        <property name="whatsThis" >
         <string>Table of the currently available layersets</string>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text" >
          <string>FileName</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QToolButton" name="toolNew" >
        <property name="toolTip" >
         <string>New</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/new.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="toolCopy" >
        <property name="toolTip" >
         <string>Clone</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/copy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QToolButton" name="toolDelete" >
        <property name="toolTip" >
         <string>Delete</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="pbnImport" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="pbnExport" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>71</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pbnClose" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pbnApply" >
     <property name="whatsThis" >
      <string>Apply the changes that have just been made to these layerset details</string>
     </property>
     <property name="text" >
      <string>Apply</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbnClose</sender>
   <signal>clicked()</signal>
   <receiver>LaPlantManagerBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>510</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel" >
     <x>164</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
